sopel/modules/admin.py

```diff
@@ -44,6 +44,27 @@ def setup(bot):
     bot.config.define_section('admin', AdminSection)
 
 
+class InvalidSection(Exception):
+    def __init__(self, section):
+        super(InvalidSection, self).__init__(self)
+        self.section = section
+
+    @property
+    def message(self):
+        return 'Section [{}] does not exist.'.format(self.section)
+
+
+class InvalidSectionOption(Exception):
+    def __init__(self, section, option):
+        super(InvalidSectionOption, self).__init__(self)
```

Comment:
Just make sure to stay inside the line length limit (which is complicated to do with Github suggestions):

```suggestion
        super(InvalidSectionOption, self).__init__(self, 'Section [{}] does not have option \'{}\'.'.format(section, option))
```

Reply:
Good thing we don't enforce the line length limit (Yet™) then :smirk:

Reply:
Much more elegant!

Done.